pkg/validation/validation.go

```diff
@@ -163,3 +165,19 @@ func nodeGroupMachineConfigPoolSelector(nodeGroups []nropv1.NodeGroup) error {
 
 	return nil
 }
+
+func MultipleMCPsPerTree(annot map[string]string, trees []nodegroupv1.Tree) error {
+	multiMCPsPerTree := annotations.IsMultiplePoolsPerTreeEnabled(annot)
+	klog.InfoS("Deprecation notice: support for multiple pools matches per node group will soon be disabled", "annotation", annotations.MultiplePoolsPerTreeAnnotation, "enabled", multiMCPsPerTree)
```

Comment:
this is good in theory, but in practice will spam the logs. Better to omit it, because doing at high verbosity levels seems pointless. We will depend on the Degraded condition to observe the behavior.